app/views/landing_page/_search_filter.html.erb

```diff
@@ -20,7 +20,7 @@
   <%= f.hidden_field :pension_pot_size, value: SearchForm::ANY_SIZE_VALUE %>
 
   <div class="search-filter--container">
-    <div class="l-results__filter">
+    <div class="l-results__filter" role="group" aria-labelledby="find_retirement_adviser_heading"">
```

Comment:
This has an extra opening speech mark at the end. 

`<div class="l-results__filter" role="group" aria-labelledby="find_retirement_adviser_heading"">` --> `<div class="l-results__filter" role="group" aria-labelledby="find_retirement_adviser_heading">`